contract/enterprise/validate.go

```diff
@@ -75,7 +75,7 @@ func ValidateEnterpriseTx(tx *types.TxBody, sender *state.V,
 		if err := checkArgs(context, &ci); err != nil {
 			return nil, err
 		}
-		key := genKey([]byte(context.Args[0]))
+		key := []byte(context.Args[0])
```

Comment:
I did not know that the function was used in other places. Above it is not converting to uppercase. Maybe the function can be bring back

Reply:
i check all key converting to uppercase in validate and execute. It's okay to revert, but it would be better to create unit tests for this.